pages/agent/v3/cli_lock.md

```diff
@@ -0,0 +1,29 @@
+# `buildkite-agent lock`
+
+The Buildkite Agent's `lock` subcommands provide the ability to coordinate multiple concurrent builds on the same host that access shared resources.
+
+>🛠 Experimental feature
+> The agent-api experiment must be enabled to use the `lock` command. To enable the agent-api experiment, include the `--experiment=agent-api` flag when starting the agent. 
```

Comment:
I set you up for a linting failure, sorry!

```suggestion
> The agent-api experiment must be enabled to use the `lock` command. To enable the agent-api experiment, include the `--experiment=agent-api` flag when starting the agent.
```

Reply:
Done.